### ISSUE TITLE
Windows gitflow install link has malware

### ISSUE THREAD

**Issue by riverfrot** [nvie/gitflow#6477]
![image](https://user-images.githubusercontent.com/47625722/178471624-4aa2f674-72b5-4a7f-8fb9-2d2cf9179b54.jpeg)

There link has malware program
So please change that link
And change permission to write at windows install.wiki file

**Comment by hmorato** [comment]
The user who uploaded the malware has been reported and banned. The download link is broken now. However, the issue about the permissions to edit the wiki remains open.

**Comment by amarc-sudo** [comment]
Yes but the malware continues to spread because once installed on a computer it will hack the GitHub account of the person and rename it to toxor4 and republish the same malware

**Comment by nvie** [comment]
Closing all issues as I'm archiving this repo. git-flow lives on at [git-flow-next](https://github.com/gittower/git-flow-next) if you're still using it. Thanks for reporting!
